### ISSUE TITLE
Registration tokens are not backuped, so they don't survive upgrade

### ISSUE THREAD

**Issue by tauceti2** [mitreid-connect/OpenID-Connect-Java-Spring-Server#1319]
Hi,

We did the upgrade of the MitreID to the latest version 1.3.1 from 1.2.6. The issue is that registration tokens were not transferred to the new version, so admins cannot access their clients.

Is there a way how to transfer registration tokens?

Cheers,

Michal P.

**Comment by jricher** [comment]
Did you follow the upgrade directions? https://github.com/mitreid-connect/OpenID-Connect-Java-Spring-Server/wiki/Upgrading

**Comment by tauceti2** [comment]
Yes. We examined the exported JSON, but it doesn't contain registration tokens.
